examples/simple/main.tf

```diff
@@ -51,14 +52,35 @@ data "aws_iam_policy_document" "s3_allow_ses_puts" {
   }
 }
 
-module "ses_bucket" {
-  source  = "trussworks/s3-private-bucket/aws"
-  version = "~> 2"
+resource "aws_s3_bucket" "temp_bucket" {
+  bucket        = var.ses_bucket
+  acl           = "private"
+  force_destroy = true
+  policy        = data.aws_iam_policy_document.s3_allow_ses_puts.json
+
+  dynamic "logging" {
+    for_each = [1]
+    content {
+      target_bucket = module.s3_logs.aws_logs_bucket
+      target_prefix = "s3/${var.ses_bucket}/"
+    }
+  }
+}
```

Comment:
What is the point of a dynamic logging element that looks like it only has a single element in it? Wondering if I'm missing something.

Reply:
I pulled it from the previously mentioned module so I'm not too sure. I was gonna change it to not use the for-each but I wasn't sure how to format this. Happy to do so with some help.

Reply:
Found the correct way to do this with a single instance. I'll go ahead and implement that.

Extra context that I just remembered: The for-each was there as part of a conditional of creating this logging stuff or not.